heudiconv/tests/test_bids.py

```diff
@@ -1116,3 +1126,32 @@ def test_BIDSFile():
     # -for an existing entity, you can overwrite it with "set":
     my_bids_file.set('echo', '2')
     assert my_bids_file['echo'] == '2'
+
+
+@pytest.mark.skipif(not have_datalad, reason="no datalad")
+def test_ME_mag_phase_conversion(tmpdir, subID='MEGRE', heuristic='bids_ME'):
+    """ Unit test for the case of multi-echo GRE data with
+    magnitude and phase.
+    The different echoes should be labeled automatically.
+    """
+    tmpdir.chdir()
+    tmppath = tmpdir.strpath
+    try:
+        datadir = fetch_data(tmppath, "dicoms/velasco/{subID}")
+    except IncompleteResultsError as exc:
+        pytest.skip("Failed to fetch test data: %s" % str(exc))
```

Comment:
note -- this line is "covered" which means that the test just skipped... I think I might see the reason